.travis.yml

```diff
@@ -4,14 +4,14 @@ branches:
      - unstable
 
 language: objective-c
-osx_image: xcode7
+osx_image: xcode7.2
```

Comment:
There is a bug in travis CI that causes the build and tests to freeze if xcode version is 7.3 and iOS version is 9.3. Till the bug is fixed we'll have to downgrade to 7.2 and iOS 9.2